### ISSUE TITLE
Delayed Job error on while exporting content

### ISSUE THREAD

**Issue by lightalloy** [thepracticaldev/dev.to#2580]
Sometimes `ExportJob` fails with the following error:

```
--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper
    job_data:
      job_class: ExportContentJob
      job_id: some_id
      provider_job_id: 
      queue_name: export_content
      priority: 
      arguments:
      - id
      executions: 0
      locale: en

Last Error
    Toggle full message

    undefined method `new' for Articles:Module
    /app/app/services/exporter/service.rb:21:in `block in export'
    /app/app/services/exporter/service.rb:20:in `each'
    /app/app/services/exporter/service.rb:20:in `export'
    /app/app/jobs/export_content_job.rb:6:in `perform'
```
This is probably related to #2031
Inside the `Exporter::Service` there is a reference to `Articles` , which is resolved to an `Articles` module. But it should be resolved to `Exporter::Articles`.

**Comment by rhymes** [comment]
Aahhhh great find! Now I understand why I can't export from the website. 

These two file create the `Articles` module: https://github.com/thepracticaldev/dev.to/blob/master/app/jobs/articles/score_calc_job.rb and https://github.com/thepracticaldev/dev.to/blob/master/app/jobs/articles/bust_cache_job.rb

😭

**Comment by lightalloy** [comment]
@rhymes that's true. But the `Articles` module is top-level, and the `Exporter::Articles` is namespaced, so probably the issue can be fixed without renaming, but I need to have a closer look at it to come up with an optimal solution.

**Comment by rhymes** [comment]
@lightalloy thank you!

**Comment by lightalloy** [comment]
Closed by #2612
